Updated FAQs with summary sentence descriptions (#545)
cisagov/getgov-home: pages/help/help_faq.md

Get more help [adding or removing domain managers](../domain-management/#add-a-domain-manager-to-your-gov-domain).

## Why don't I see a domain when I sign in to the registrar? {#do-not-see-my-domain}

> Make sure you are signed into Login.gov with the correct email address associated with that domain.

Reviewer: @Katherine-MN @waiwanwong  I think this should be "signed in to Login.gov" (instead of "into") but I'll defer to you all.

> Make sure you are signed into Login.gov with the correct email address associated with that domain.

Reviewer 2: ```suggestion
Make sure you are signed in to Login.gov with the correct email address associated with that domain.
```

If you’ve tried the above recommendations and still don’t see the domain, [contact us](../../contact/).

## A domain will expire soon. How can I renew it? {#renew-domain}

> Learn [how to renew a domain](https://get.gov/help/domain-management/#renew-an-expiring-or-expired-gov-domain){target="_blank"}.

Reviewer 3: ```suggestion
Learn [how to renew a domain](../help/domain-management/#renew-an-expiring-or-expired-gov-domain){target="_blank"}.
```

Reviewer 3: Using relative links ensures preview builds point to the preview version, not their canonical home.

You can renew a domain starting 60 days before expiration. 

Email notifications for expiring domains are sent to domain managers 30 days, 7 days, and 1 day before the expiration date.

Any domain manager can renew a domain. After renewal, the domain is registered for another year from the expiration date.

## What happens if I don’t renew a domain? {#dont-renew}

> If a domain expires and has DNS name servers_: We’ll make an extensive effort to contact your organization before placing the domain on hold or deleting it. The domain can be renewed until our team deletes it.

Reviewer 2: ```suggestion
**If a domain expires and has DNS name servers:** We’ll make an extensive effort to contact your organization before placing the domain on hold or deleting it. The domain can be renewed until our team deletes it.
```

Reviewer 2: To apply bold formatting, insert a double asterisk before and after the phrase.

> If a domain expires and does not have DNS name servers_: The domain will be deleted immediately. 

Reviewer 2: ```suggestion
**If a domain expires and does not have DNS name servers:** The domain will be deleted immediately. 
```
